pkg/controller/controller.go

```diff
@@ -285,64 +283,64 @@ func (c *controller) getServiceClassPlanAndServiceBroker(instance *v1alpha1.Serv
 // getServiceClassPlanAndServiceBrokerForServiceInstanceCredential is a sequence of operations that's
```

Comment:
Will you expand on the godoc comment here to note that the method updates the status of the binding if one of th eprecondition checks implemented in the method fails?  Same thing with the instance one.